src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/SaveOidcngResourceServerEntityCommand.php

```diff
@@ -19,6 +19,7 @@
 namespace Surfnet\ServiceProviderDashboard\Application\Command\Entity;
 
 use InvalidArgumentException;
+use phpDocumentor\Reflection\Types\Boolean;
```

Comment:
Please remove this import. I guess it was added by mistake?

Reply:
it was